docs/standards/universal-profile/07-lsp9-vault.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_label: 'LSP9 - Vault'
+sidebar_position: 7
+---
+
+# LSP9 - Vault
+
+:::info Standard Document
+
+[LSP9 - Vault](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-9-Vault.md)
+
+:::
+
+## Introduction
+
+Using the KeyManager with the UniversalProfile will enable third parties to execute through your profile given specific permissions, but this will not eliminate the risk of operating maliciously with your data and belongings.
```

Comment:
```suggestion
Using the Key Manager with the Universal Profile will enable third parties to execute through your profile given specific permissions, but this will not eliminate the risk of operating maliciously with your data and belongings.
```